pkg/scheduler/eventhandlers.go

```diff
@@ -46,30 +46,10 @@ import (
 	"k8s.io/kubernetes/pkg/scheduler/util/assumecache"
 )
 
-func (sched *Scheduler) onStorageClassAdd(obj interface{}) {
```

Comment:
This PR is marked as draft because I wanted to create a separate PR for this change, which I found during the change for this PR.

Reply:
https://github.com/kubernetes/kubernetes/pull/127798